src/guide/composition-api-setup.md

```diff
@@ -103,7 +103,7 @@ We'll explain the role of `expose` shortly.
 
 ## Accessing Component Properties
 
-When `setup` is executed, the component instance has not been created yet. As a result, you will only be able to access the following properties:
+When `setup` is executed, although the component instance has already been created, you will only be able to access the following properties:
```

Comment:
**suggestion**: I would probably go even further and reduce it to:
```suggestion
When `setup` is executed, you will only be able to access the following properties:
```

Reply:
this is a good suggestion